apps/segment/src/connectors/auth.ts

```diff
@@ -0,0 +1,7 @@
+export const validateToken = async (token: string) => {
+  const response = await fetch(`https://api.segmentapis.com/echo?message=valid`, {
+    headers: { Authorization: `Bearer ${token}` },
+  });
+
+  return response.status;
```

Comment:
```suggestion
if (!response.ok) {
  throw new SegmentError('Could not validate token', { response })
}
```

No need to returns true, if the function don't throw, the token is valid.

Reply:
Done

Reply:
I still seeing the response.status